src/System.Dynamic.Runtime/ref/System.Dynamic.Runtime.csproj

```diff
@@ -7,7 +7,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Dynamic.Runtime.cs" />
-    <Compile Include="System.Dynamic.Runtime.Manual.cs" />
+    <!-- ToDo: Remove this P2P Reference once packages are updated -->
```

Comment:
I am wondering who and when is going to follow up on this ToDo. 
Could you enter an issue to track this?